### ISSUE TITLE
Easiest DUPE with Create mod + Video

### ISSUE THREAD

**Issue by Proxwian** [P3pp3rF1y/SophisticatedStorage#449]
Unfortunately, when you place double chest from Sophisticated on rotational Contraption, it converts into two chests with the same items.

Please add config option to prevent double chests to appear from mod

**Comment by Proxwian** [comment]
https://github.com/user-attachments/assets/3928f140-0c36-4f7a-a74c-00cda91bb68d

**Comment by Proxwian** [comment]
Anyone?

**Comment by P3pp3rF1y** [comment]
Yes this is possible with create, there's really not much I can do about that other than working closely with create team to allow adding compatibility. This is not the only thing that can happen and in fact there's possibility for item loss as well so purely banning double chests isn't going to fix it. 
I will keep this open for potential compat in the future.

**Comment by Pqvel1l** [comment]
@P3pp3rF1y did you fix it? If no, u can just add for all blocks create TAG : create:non_movable
and all your blocks do not movable by Create contraptions.

**Comment by P3pp3rF1y** [comment]
> @P3pp3rF1y did you fix it? If no, u can just add for all blocks create TAG : create:non_movable and all your blocks do not movable by Create contraptions.

I haven't "fixed" this no (the proper fix would be more on create side) and I don't want my blocks to me unmovable, but good to know that modpack makers can add that tag if they want to prevent this.

**Comment by P3pp3rF1y** [comment]
We now have a proper full featured Sophisticated Storage Create compat in https://www.curseforge.com/minecraft/mc-mods/sophisticated-storage-create-integration for 1.20 and 1.21 so I am closing this
